yaksh/templates/yaksh/question.html

```diff
@@ -242,7 +259,7 @@ <h4 class="modal-title" id="myModalLabel">File not selected</h4>
   editor.setValue('{{ last_attempt|escape_quotes|safe }}')
 
   function reset_editor() {
-      editor.setValue('{{ last_attempt|escape_quotes|safe }}');
+      editor.setValue('');
```

Comment:
For now reverted back to the original code.
The problem is if I write a code and submit and then want to reset, then the reset will show the same code.